machines/guides-examples/multi-container-machines.html.markerb

```diff
@@ -166,14 +166,18 @@ flyctl machine run --machine-config cli-config.json \
 Alternatively, you can make a direct API call using `curl`:
 
 ```bash
-curl -X POST "https://api.fly.io/v1/apps/your-app-name/machines" \
+curl -X POST "https://api.machines.dev/v1/apps/your-app-name/machines" \
   -H "Authorization: Bearer your-api-token" \
   -H "Content-Type: application/json" \
   -d @machine-config.json
 ```
 
 Refer to the [Machines API documentation](https://docs.machines.dev/#tag/machines) for more details.
 
+<div class="callout">
```

Comment:
@Lucais11 I added your suggested note as a callout. If you don't think it needs a callout, I'll remove that formatting.